hotnews/index.html

```diff
@@ -1,37 +1,107 @@
 <html>
-  <head>
-    <title>HotNews.ro</title>
-    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
 
-    <script src="https://cdn.tailwindcss.com"></script>
-    <script>
-      tailwind.config = {
-        theme: {
-          extend: {
-            colors: {
-              hotblue: '#2145ac',
-              hotgreen: '#3f9863',
-              hotred: '#eb3223'
-            }
+<head>
+  <title>HotNews.ro</title>
+  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon.png">
+  <link rel="stylesheet" href="hotnews.css">
+
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script>
+    tailwind.config = {
+      theme: {
+        fontSize: {
+          font: '15px',
+          font2: '18px',
+          font3: '14px'
+        },
+        fontFamily: {
+          sans: ["Inter", "sans-serif"]
+        },
+        minheight: '90px',
+        extend: {
+          padding: {
+            paddingleft: '22px'
+          },
+          width: '1349px',
+          height: {
+            height: '90px',
+            height2: '52px',
+            height3: '22px',
+            height4: '50px'
+          },
+          colors: {
+            hotblue: '#2145ac',
+            hotgreen: '#3f9863',
+            hotred: '#eb3223',
+            hotblack: '#393939',
+            hotEconomie: '#ed5c43',
```

Comment:
Please use color names instead of section names when defining colours.
You can use this website for naming colors,
eg. for this one, https://chir.ag/projects/name-that-color/#ED5C43 , the actual color name is Burnt Sienna
so you could use **hotBurntSienna**